frontend/src/stimulus/controllers/dynamic/work-packages/relations-tab/scroll.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class ScrollController extends Controller {
+  connect() {
+    this.waitForRenderAndAct();
+  }
+
+  waitForRenderAndAct() {
+    const element = document.querySelector('[data-scroll-to="true"]');
```

Comment:
I'd recommend to use a stimulus target instead of a normal data attribute. The value is crutial for this controller to work, so it should be an explicit target imho.